scripts/post-update.sh

```diff
@@ -327,6 +327,20 @@ if (( $PRE_0_0_19 > 0 )); then
     # The user should apply the default parameters if he wishs to use all features available
 fi
 
+# Check pre-0.0.21 to merge our updated (0.0.17...0.0.20) default mavproxy configuration
+# with any existing user configuration
+PRE_0_0_21=$(( git rev-list --count --left-right 0.0.21...revert-point || echo 0 ) | cut -f1)
+
+if (( $PRE_0_0_21 > 0 )); then
+    USER_MAVPROXY_PARAMS=$HOME/mavproxy.param
+    if [ -e $USER_MAVPROXY_PARAMS ]; then
+        sed -i '/--streamrate*/d' $USER_MAVPROXY_PARAMS
+        sed -i '$ a --streamrate 10' $USER_MAVPROXY_PARAMS
+        sed -i '$ a --out udpout:localhost:9002' $USER_MAVPROXY_PARAMS
+        sed -i '$ a --out udpin:0.0.0.0:14660' $USER_MAVPROXY_PARAMS
```

Comment:
What happens if these lines already exist in the param file ?

Reply:
unlikely but not impossible, and then I don't know what mavproxy would do, but probably just act 2x for duplicate configurations (1x on each occurance). updated to handle it

Reply:
@jaxxzer updated the code to delete the lines if they are already there, check the current diff